doc/examples/streamline_length.py

```diff
@@ -139,7 +139,7 @@ def simulated_bundles(no_streamlines=50, n_pts=100):
    :align: center
 
    Initial bundle (down), downsampled at 12 equidistant points (middle),
-   downsampled not equidistantly (up).
+   downsampled not equidistant (up).
```

Comment:
I agree with @jhlegarreta that "not equidistantly" is technically acceptable, but it's an eye sore. How about the following? 

```suggestion
   downsampled with points that are not equidistant (up).
```